examples/with-cxs/pages/_document.js

```diff
@@ -0,0 +1,25 @@
+import Document, { Head, Main, NextScript } from 'next/document'
+import cxs from 'cxs'
+
+export default class MyDocument extends Document {
+  static async getInitialProps ({ renderPage }) {
+    const page = renderPage()
+    let style = cxs.getCss()
```

Comment:
@jxnblk is there a `cxs.reset` command or similar, that would ensure to clear _dirty_, _unclaimed_ state (for example if `renderPage` throws here?